git-hub

```diff
@@ -87,14 +97,142 @@ def push():
             other_user = data["user"]
             pr = data["pr"]
             branch = data["branch"]
-            p = subprocess.Popen(["git", "push", other_user,
-                                 f'{pr}:{branch}'], stdout=subprocess.PIPE)
+            p = subprocess.Popen(["git", "push", other_user, f'{pr}:{branch}'], stdout=subprocess.PIPE)
             p.stdout.read()
     except FileNotFoundError as e:
         print(e)
         print("This command must be used after git hub pr.")
         sys.exit(1)
 
+def search(keyword, user, comment, number, branch):
+    """Searches open and closed pull request comments for specified keyword.
+    Opens pull-requests.toml file in .git folder to fetch pull requests.
+
+    Parameters
+    ----------
+    keyword : string
+        Searches if any part of user, branch, comment, or number match
+    user : string
+        Search by PR username
+    comment : string
+        Search by PR comment
+    number : string
+        Search by PR number
+    branch : string
+        Search by PR branch
+    """
+    try:
+        open_prs = False
+        closed_prs = False
+        pathToGit = './'
+        while not any(fname == '.git' for fname in os.listdir(pathToGit)):  #if not in directory with .git, keep going back to find file
```

Comment:
Use glob.glob

Reply:
Perhaps walkpath?

Reply:
Sorry, not sure I understand what you mean, could you elaborate?